### DISCUSSION TITLE
Line spacing does not work

### DISCUSSION THREAD

**Discussion by petkrein** [gonzalezreal/swift-markdown-ui#247] [Q&A] [answered]
Line spacing doesn't seem to work. 
Neither '.lineSpacing(1.2)' nor via your custom view modifier '.relativeLineSpacing(.em(0.9))'.

It would be nice if you show me how to manipulate the line spacing.

**Example 1:**
```
 Markdown(self.content)
       .lineSpacing(1.5)
```

![Bildschirmfoto 2023-08-02 um 17 57 00](https://github.com/gonzalezreal/swift-markdown-ui/assets/18345890/77d3362b-1ae4-4a4f-9bb2-5fd4357f9fcb)

**Example 2:**

```
Markdown(self.content)
       .relativeLineSpacing(RelativeSize.em(0.9))
```

![Bildschirmfoto 2023-08-02 um 17 47 39](https://github.com/gonzalezreal/swift-markdown-ui/assets/18345890/7a7716ae-8b0a-4918-98e8-f3b41b46358d)

**Version information**
- MarkdownUI: [2.0.0]
- OS: [iOS 16.4, macOS 13.5]
- Xcode: [14.3.1]

**Comment by gonzalezreal** [comment, marked as answer]
Hi @petkrein,

Please consider that a `Markdown` view comprises multiple subviews (usually text, images, and containers), so the `lineSpacing` modifier might not work as you expect when you apply it directly to it.

Instead, you can make these customizations at a block level (paragraph, heading, etc.), which is very powerful because you can customize the line spacing, the top and bottom margins, and many other things.

[This post](https://gonzalezreal.github.io/2023/02/18/better-markdown-rendering-in-swiftui.html#styling-blocks) will give you all the details.

Since this is not an issue, I am moving it to a discussion in case you want to follow up with more questions.
